clr-installer/main.go

```diff
@@ -289,6 +290,14 @@ func execute(options args.Args) error {
 	if options.SwupdFormat != "" {
 		md.SwupdFormat = options.SwupdFormat
 	}
+	if options.SwupdVersion != "" {
+		version, err := strconv.ParseUint(options.SwupdVersion, 10, 32)
```

Comment:
"latest" is a valid input for swupd's version argument. Can you add check for this case.

Reply:
You should set the version to 0 when latest is provided

Reply:
@jwakre Good catch.  We only allow an unsigned in the YAML so I was following the same convention for the command line. But now allow `latest`.
Force pushed a fix.